src/components/TabSystem.js

```diff
@@ -98,6 +100,26 @@ const TabSystem = (props) => {
     });
   }
 
+  if (window.location.hash.length && !toastError.isActive("errorToast")) {
+    // eslint-disable-next-line no-restricted-globals
+    history.pushState("", "", window.location.origin);
+
+    toastError({
+      id: "errorToast",
```

Comment:
Fijate si te queda mas lindo el diseño así, para mantener consistencia con el otro toast. Si no, ya fue

```suggestion
      id: "errorToast",
      position: "bottom-start"
```